package.json

```diff
@@ -1,93 +1,84 @@
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@fortawesome/fontawesome-svg-core": "^6.5.2",
+    "@fortawesome/free-solid-svg-icons": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "@mui/icons-material": "^5.15.14",
+    "@mui/material": "^5.15.14",
     "autoseg": "^0.0.12",
-    "clamp": "^1.0.1",
-    "color-alpha": "^1.0.4",
-    "get-image-data": "^3.0.1",
-    "material-survey": "^2.1.0",
-    "mmgc1-cpp": "^1.0.50",
-    "moment": "^2.23.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-full-screen": "^0.3.1",
+    "classnames": "^2.5.1",
+    "color-alpha": "^2.0.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.30.0",
+    "monaco-editor": "^0.47.0",
+    "react-full-screen": "^1.1.1",
     "react-hotkeys": "^2.0.0",
-    "react-json-view": "^1.19.1",
-    "react-markdown": "^4.0.6",
-    "react-material-workspace-layout": "^1.0.10",
-    "react-monaco-editor": "^0.25.1",
+    "react-markdown": "^9.0.0",
+    "react-monaco-editor": "^0.55.0",
     "react-remove-scroll": "^2.0.4",
-    "react-select": "^3.0.8",
-    "react-syntax-highlighter": "^12.2.1",
-    "react-use": "^13.27.0",
+    "react-select": "^5.8.0",
+    "react-syntax-highlighter": "^15.5.0",
+    "react-use": "^17.5.0",
     "react-use-measure": "^2.0.0",
     "seamless-immutable": "^7.1.4",
-    "shallow-equal": "^1.2.1",
-    "storybook": "^5.3.14",
-    "styled-components": "^5.2.1",
     "transformation-matrix-js": "^2.7.6",
-    "use-event-callback": "^0.1.0",
-    "use-key-hook": "^1.3.0"
+    "tss-react": "^4.9.6",
+    "use-event-callback": "^0.1.0"
+  },
+  "devDependencies": {
+    "@rollup/plugin-replace": "^5.0.5",
+    "@types/lodash": "^4.17.0",
+    "@types/node": "^20.12.4",
+    "@types/react": "^18.2.66",
+    "@types/react-dom": "^18.2.22",
+    "@types/react-select": "^5.0.1",
+    "@types/react-syntax-highlighter": "^15.5.11",
+    "@types/rollup-plugin-peer-deps-external": "^2.2.5",
+    "@types/seamless-immutable": "^7.1.4",
+    "@vitejs/plugin-react": "^4.2.1",
+    "cpy-cli": "^5.0.0",
+    "cross-env": "^7.0.3",
+    "gh-pages": "^2.0.1",
+    "prettier": "^2.5.1",
+    "react-github-btn": "^1.1.1",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "typescript": "^5.2.2",
+    "vite": "^5.2.7",
+    "vite-plugin-dts": "^3.8.1",
+    "vite-plugin-node-polyfills": "^0.21.0",
+    "vite-tsconfig-paths": "^4.3.2"
   },
   "peerDependencies": {
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
   },
-  "homepage": "/react-image-annotate",
   "repository": {
     "type": "git",
-    "url": "https://github.com/UniversalDataTool/react-image-annotate.git"
+    "url": "git+https://github.com/idapgroup/react-image-annotate.git"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "storybook": "start-storybook -p 9090 -s public",
```

Comment:
hmm removing storybook means removing a lot of tests, is there any way to keep the stories and upgrade storybook? I'm worried that we've basically lost all of our tests.

If storybook isn't your vibe, we could use react cosmos or ladle, but I think keeping the stories (which are essentially tests) will help me make sure that nothing broken.